docs/specs/openapi.json

```diff
@@ -8605,6 +8725,64 @@
                 },
                 "description": "The ordering to apply to the report. Default is ascending order for the data."
             },
+            "ReportAWSEC2ComputeOrdering": {
+                "type": "object",
+                "properties": {
+                    "cost": {
+                        "type": "string",
+                        "enum": [
+                            "asc",
+                            "desc"
+                        ]
+                    },
+                    "usage": {
+                        "type": "string",
+                        "enum": [
+                            "asc",
+                            "desc"
+                        ]
+                    },
+                    "account": {
+                        "type": "string",
+                        "enum": [
+                            "asc",
+                            "desc"
+                        ]
+                    },
+                    "operating_system": {
+                        "type": "string",
+                        "enum": [
+                            "asc",
+                            "desc"
+                        ]
+                    },
+                    "resource_id": {
+                        "type": "string",
+                        "enum": [
+                            "asc",
+                            "desc"
+                        ]
+                    },
+                    "instance_name": {
+                        "type": "string",
+                        "enum": [
+                            "asc",
+                            "desc"
+                        ]
+                    },
+                    "instance_type": {
```

Comment:
Why is instance_type not a valid filter? (especially since we can order by it)

Reply:
That's a good question but most likely because its not in the list of fields to filter by in the design doc.